hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/BusyRegionSplitPolicy.java

```diff
@@ -60,6 +60,12 @@
   private long blockedRequestCount;
   private float blockedRate;
 
+  @Override
+  public String toString() {
+    return "BusyRegionSplitPolicy{" + "maxBlockedRequests=" + maxBlockedRequests + ", minAge="
```

Comment:
Late to the party, but how about using `ToStringBuilder` from apache commons? `SHORT_PREFIX_STYLE` is 👌